doc/debian/jitsi-meet/jitsi-meet.example

```diff
@@ -40,8 +40,10 @@ server {
     error_page 404 /static/404.html;
 
     gzip on;
-    gzip_types text/plain text/css application/javascript application/json;
+    gzip_types text/plain text/css application/javascript application/json image/x-icon application/octet-stream;
```

Comment:
Is anything actually served with the application/octet-stream mime type?

Reply:
Maybe wasm?

Reply:
The WAV files are application/octet-stream (at least on my system) and they compress nicely. I will add WASM (application/wasm), it compresses around 12% in my test.